solutionbox/code_free_ml/mltoolbox/code_free_ml/data/analyze_data.py

```diff
@@ -29,6 +29,8 @@
 import textwrap
 
 from tensorflow.contrib import lookup
+from tensorflow.contrib.slim.python.slim.nets.inception_v3 import inception_v3
```

Comment:
Do we know what the plans from the TF side are for bringing this out of contrib?